.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     working_directory: ~/dlang.org
     docker:
-      - image: circleci/node:4.8.2
+      - image: cimg/node:lts
```

Comment:
I'm pretty sure we're not using `node` anywhere in this repository.

Reply:
I'll try it with `cimg/base`.